ckan/logic/action/get.py

```diff
@@ -2533,14 +2533,17 @@ def package_activity_list(
 
     :param id: the id or name of the package
     :type id: string
-    :param offset: where to start getting activity items from
```

Comment:
@wardi was your idea to remove support for `offset` pagination entirely or keep it at the API level and just not use it from the UI?

Reply:
Do we know how many users of the activity API we have?

My guess is we would want to keep it for compatibility, at least for a while.

Reply:
The `offset` para was restored. Works well in conjunction with `after` and `before` (and not used for pagination).
Also, some test were added